native_client/ctcdecode/ctc_beam_search_decoder.cpp

```diff
@@ -96,24 +96,52 @@ DecoderState::next(const double *probs,
         if (full_beam && log_prob_c + prefix->score < min_cutoff) {
           break;
         }
+        if (prefix->score == -NUM_FLT_INF) {
+          continue;
+        }
```

Comment:
This seems unrelated to the rest of the PR.

Reply:
I had crashes without it in my early tests, I will try it again.